fastlane/lib/fastlane/actions/setup_ci.rb

```diff
@@ -108,7 +113,7 @@ def self.is_supported?(platform)
       def self.example_code
         [
           'setup_ci(
-            provider: "travis"
+            provider: "circleci"
```

Comment:
Any reason to change the example ?

Reply:
I changed this to to prevent false assumptions that passing `travisci` changed the behaviour of this action, which it doesn't.